docs/source/reference/debugging.rst

```diff
@@ -98,7 +98,7 @@ Specifically dumped are:
    Erasable Arguments, Things that are erased.
    Detaggable argument types,
    Specialised arguments,
-   Inferrable arguments,
+   Inferable arguments,
```

Comment:
Inferrable (with 2 'r's) is also correct, and the compiler uses it, so could you change this (and the other uses) back to inferrable?